portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,67 @@
+                                    "Created a device that prevents false alarms triggered by the existing system with a button embedded in the firefighters' gloves, which signals that they are conscious",
+                                    "New system envisioned to use infraed signals to communicate between button and device --prototype was built using an Arduino and a motor that when triggered, causes rotatable arm to deactivate PASS alarm",
+                                    "Conducted research with input from Toronto Fire Department to ensure safety standards were met"],
+        "ESEC Website": ["Created and maintained a multi-page website for the Engineering Science Education Conference using WordPress and custom html/css",
+                                    "Analyzed previous legacy code and utilized it as boilerplate infrastructure while updating source-code as required",
+                                    "Applied concepts of responsive web design in order to make page accessible and user-freindly across all devices"], 
+        "Matboard Construction Bridge Project": ["Constructed a ~1.5m long bridge using matboard only",
+                                    "Applied knowledge of stress, shear, and buckling to optimize bridge load",
+                                    "Acheived maximum load of 800N before failure"]                             
+        };
+      var TheInnerHTML ="";
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+      for (project in projects) {
+          TheInnerHTML += "<h5>"+project+"</h5>";
+          TheInnerHTML += "<ul>";
+          for (i = 0; i < projects[project].length; i++) {
+              TheInnerHTML += "<li style=\"list-style-type:circle\">"+projects[project][i]+"</li>";
+          }
+          TheInnerHTML += "</ul>";
+    }
+    document.getElementById("projectContent").innerHTML = TheInnerHTML;
+}
+
+//play a game of two truths and one lie
+function playGame() {
+    const truths = ['I play the piano.', 
+                    'I don\'t like pineapples on pizza.',
+                    'I have never been skydiving before.', 
+                    'I am hungry.', 
+                    'I visited my friend in Manchester last Christmas.', ]; 
+    const lies = ['My favourite subject in high school was Calculus.',  
+                    'I have two brothers.',
+                    'I was born in the year of the tiger.']; 
+
+
+    // generate two truths and one lie
+    truth_1_index = Math.floor(Math.random() * truths.length);
+    do {
+        truth_2_index = Math.floor(Math.random() * truths.length);
+    }while (truth_1_index === truth_2_index);
+    
+    lie_index = Math.floor(Math.random() * lies.length);
+    twoTruthsOneLie = [truths[truth_1_index], truths[truth_2_index], lies[lie_index]];
+    twoTruthsOneLie.sort() //sort the list so that order is randomized
```

Comment:
this is a funny comment. Isn't sorted by definition not randomized?

Consider actually randomizing the order here.

Reply:
Done :)